src/Akka.Persistence.Hosting/AkkaPersistenceHostingExtensions.cs

```diff
@@ -226,14 +363,21 @@ public static AkkaConfigurationBuilder WithInMemoryJournal(
             string journalId = "inmem",
             bool isDefaultPlugin = true)
         {
-            builder.WithJournal(journalId, journalBuilder);
```

Comment:
This method is obsolete. We should probably re-write this to use a `JournalOptions` implementation but that's a bit out of scope for this PR.